reduction/line_imaging.py

```diff
@@ -144,15 +171,14 @@
             logprint("Computing residual image statistics for {0}".format(lineimagename), origin='almaimf_line_imaging')
             ia.open(lineimagename+".residual")
             stats = ia.statistics(robust=True)
+            # to figure out why the coefficient is needed here. Generally, this method applying to the entire image plane
+            # over all channels results in a higher rms than the actual value.
```

Comment:
The coefficient is the multiplicative factor between the raw MAD and the standard deviation.  See https://en.wikipedia.org/wiki/Median_absolute_deviation